tools/gradle-plugin/build.gradle

```diff
@@ -49,6 +49,9 @@ test {
     // multiple Gradle invocations executed from the same testing JVM cause problems, so fork
     // for each test, and make sure we only run one Gradle execution in each test
     forkEvery 1
+    // For Gradle testing :(
+    jvmArgs("--add-opens=java.base/java.lang.invoke=ALL-UNNAMED",
```

Comment:
This won't work with Java < 9, hope that's okay or shall I add a version-gate?

Reply:
I think this is fine - we're only running CI with 11 and 17 anymore.